### DISCUSSION TITLE
Where is the final code evaluation?

### DISCUSSION THREAD

**Discussion by fersebas** [PaddlePaddle/PaddleSpatial#80] [BaiduKDDCup2022] [answered]
Hello, I see there are many files similar but differents. Which one is the final one that is using by AISTUDIO and leaderboard

the code in https://github.com/PaddlePaddle/PaddleSpatial/tree/main/apps/wpf_baseline_gru/kddcup22-sdwpf-evaluation/paddlepaddle ? It is updated?  Please check it because I receive better score with this than with the old one.

Thanks so much.

**Comment by ramber1836** [comment, marked as answer]
Hi, there,

Yes, the code you referred, i.e. located
under PaddleSpatial/apps/wpf_baseline_gru/kddcup22-sdwpf-evaluation/paddlepaddle,
is the latest version we adopt for online test.
The score is improved because of the updated evaluation strategy.

Thanks.

Regards,
Xinjiang Lu
